cartography/intel/aws/ec2/snapshots.py

```diff
@@ -12,13 +12,29 @@
 logger = logging.getLogger(__name__)
 
 
+@timeit
+def get_snapshots_in_use(neo4j_session: neo4j.Session, region: str, current_aws_account_id: str) -> List[str]:
```

Comment:
Please try and add an integration test for this.

Reply:
👍 Added!